config/webpack.config.test.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable */
+const path = require('path');
+const { resolve } = require('path');
+const merge = require('webpack-merge');
+const webpack = require('webpack');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+const { NamedModulesPlugin } = require('webpack');
+const StyleLintPlugin = require('stylelint-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const baseConfig = require('./webpack.config');
+const reactConfig = require('./webpack.config.react');
+/* eslint-enable */
+
+const reactToolboxVariables = {
+  'color-primary': '#0288D1',
+  'color-primary-dark': '#0288D1',
+  'button-border-radius': '3px',
+  'input-text-label-color': 'rgba(0,0,0,0.38)',
+};
```

Comment:
`reactToolboxVariables` should be defined in one place.
Now they are in in 3: here,`config/webpack.config.prod.js` and `config/webpack.config.dev.js`

I would create `config/reactToolboxVariables.js` for it.